spec/assessments/fleschReadingEaseSpec.js

```diff
@@ -1,107 +1,125 @@
-var fleschReadingAssessment = require( "../../js/assessments/readability/fleschReadingEaseAssessment.js" );
-var Paper = require( "../../js/values/Paper.js" );
+import FleschReadingAssessment from "../../js/assessments/readability/fleschReadingEaseAssessment.js";
```

Comment:
I don't know too much about the technical differences between `import` and `require`, but since we usually use `require` I'd stick with that (if you don't have any specific reasons to use `import` here).